CLI/brownfield-deployment/deploySpring.sh

```diff
@@ -0,0 +1,110 @@
+azurespring_resource_group_name=$azurespringrg
+
+echo "Enter Azure Spring cloud SKU (Enterprise or Standard): "
+read azurespringsku
+azurespring_sku=$azurespringsku
+
+echo "Enter Azure Spring cloud VNet Resource Group Name: "
+read azurespringvnetrg
+azurespring_vnet_resource_group_name=$azurespringvnetrg
+
+echo "Enter Azure Spring cloud Spoke VNet : "
+read azurespringappspokevnet
+azurespringappspokevnet=$azurespringappspokevnet
+
+echo "Enter Azure Spring cloud App SubNet : "
+read azurespringappsubnet
+azurespring_app_subnet_name='/subscriptions/'$subscription'/resourcegroups/'$azurespring_vnet_resource_group_name'/providers/Microsoft.Network/virtualNetworks/'$azurespringappspokevnet'/subnets/'$azurespringappsubnet
+
+echo "Enter Azure Spring cloud Service SubNet : "
+read azurespringservicesubnet
+azurespring_service_subnet_name='/subscriptions/'$subscription'/resourcegroups/'$azurespring_vnet_resource_group_name'/providers/Microsoft.Network/virtualNetworks/'$azurespringappspokevnet'/subnets/'$azurespringservicesubnet
+
+echo "Enter Azure Log Analytics Workspace Resource Group Name: "
+read loganalyticsrg
+loganalyticsrg=$loganalyticsrg
+
+echo "Enter Log Analytics Workspace Resource ID: "
+read workspace
+workspaceID='/subscriptions/'$subscription'/resourcegroups/'$loganalyticsrg'/providers/microsoft.operationalinsights/workspaces/'$workspace
+
+echo "Enter Reserved CIDR Ranges for Azure Spring Cloud: "
+read reservedcidrrange
+reservedcidrrange=$reservedcidrrange
+
+echo "Enter key=value pair used for tagging Azure Resources (space separated for multiple tags): "
+read tag
+tags=$tag
+
+randomstring=$(LC_ALL=C tr -dc 'a-z0-9' < /dev/urandom | fold -w 13 | head -n 1)
+azurespring_service='spring-'$randomstring #Name of unique Spring Cloud resource
+azurespring_appinsights=$azurespring_service
+azurespring_resourceid='/subscriptions/'$subscription'/resourceGroups/'$azurespring_resource_group_name'/providers/Microsoft.AppPlatform/Spring/'$azurespring_service
+
+# Create Application Insights
+az monitor app-insights component create \
+    --app ${azurespring_service} \
+    --location ${location} \
+    --kind web \
+    -g ${azurespringrg} \
+    --application-type web \
+    --workspace ${workspaceID}
+
+wait    # Wait for the above command to complete
+
+if [ ${azurespringsku} == "Standard" ]; then
+    echo "Creating Azure Spring Standard"
+        az spring create \
+            -n ${azurespring_service} \
+            -g ${azurespringrg} \
+            -l ${location} \
+            --sku ${azurespringsku} \
+            --enable-java-agent true \
+            --app-insights ${azurespring_service} \
+            --app-subnet ${azurespring_app_subnet_name} \
+            --service-runtime-subnet ${azurespring_service_subnet_name} \
+            --reserved-cidr-range ${reservedcidrrange} \
+            --tags ${tags}
+        wait    # Wait for the above command to complete
+elif [ ${azurespringsku} == "Enterprise" ]; then
```

Comment:
@vinodramasubbu - Isn't this case sensitive? Wondering what the behavior would be if someone types "enterprise" in lower case